kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/log/AnsiLogger.java

```diff
@@ -0,0 +1,32 @@
+package org.arquillian.cube.kubernetes.impl.log;
+
+import org.arquillian.cube.kubernetes.api.Logger;
+
+import static org.fusesource.jansi.Ansi.Color.GREEN;
+import static org.fusesource.jansi.Ansi.Color.RED;
+import static org.fusesource.jansi.Ansi.Color.YELLOW;
+import static org.fusesource.jansi.Ansi.ansi;
+
+public class AnsiLogger implements Logger {
+
+    @Override
+    public void info(String msg) {
+        System.out.println(msg);
+    }
+
+    @Override
+    public void warn(String msg) {
+        System.out.println( ansi().fg(YELLOW).a(msg).reset() );
```

Comment:
I think that in this case it is because the AnsiLogger, but as you said we could spread to other pieces of cube.